pallets/funding/src/benchmarking.rs

```diff
@@ -986,93 +928,164 @@ mod benchmarks {
 				project_id == project_id,
 				ct_amount == bid_params.amount,
 				multiplier == bid_params.multiplier
-			}
-			.expect("Event has to be emitted");
+			};
+			assert!(maybe_event.is_some(), "Event not found");
 		}
 	}
 
 	#[benchmark]
-	fn bid(
-		// amount of already made bids by the same user. Leave y::max (10) to make the extrinsic pass
-		x: Linear<0, { T::MaxBidsPerUser::get() - 10 }>,
-		// amount of times where `perform_bid` is called (i.e how many buckets)
-		y: Linear<0, 10>,
+	fn end_auction(
+		// Accepted Bids
+		x: Linear<10, { 25 }>,
+		// Failed Bids
+		y: Linear<0, { 8 }>,
 	) {
```

Comment:
I think I don't understand this benchmark:
1. Why are we using the `{ }`?
2. Why do we use x [10, 25] and y[0, 8]? We were previously using many more accepted/rejected bids? Are you saying in this way we check for the following values and extrapolate for values > 25 and 8?

Reply:
This is because the only way for a bid to fail now is to have oversubscription, so we cannot have no accepted bids and only rejected bids. Also the way the benchmark is coded it would be a PIA to have the old ranged. This new ones work and are enough to extrapolate